golang/src/main/java/com/regnosys/rosetta/generator/golang/functions/GolangFunctionGenerator.xtend

```diff
@@ -0,0 +1,86 @@
+package com.regnosys.rosetta.generator.golang.functions
+
+import com.regnosys.rosetta.generator.golang.object.GolangModelObjectBoilerPlate
+import com.regnosys.rosetta.rosetta.RosettaCardinality
+import com.regnosys.rosetta.rosetta.RosettaNamed
+import com.regnosys.rosetta.rosetta.simple.Attribute
+import com.regnosys.rosetta.rosetta.simple.Function
+import com.regnosys.rosetta.rosetta.simple.FunctionDispatch
+import java.util.HashMap
+import java.util.List
+import java.util.Map
+import javax.inject.Inject
+
+import static com.regnosys.rosetta.generator.golang.util.GolangModelGeneratorUtil.*
+
+import static extension com.regnosys.rosetta.generator.golang.util.GolangTranslator.toGOType
+
+class GolangFunctionGenerator {
+	@Inject extension GolangModelObjectBoilerPlate
+	
+	static final String FILENAME = 'functions.go'
+		
+	def Map<String, ? extends CharSequence> generate(Iterable<RosettaNamed> rosettaFunctions, String version) {
+		val result = new HashMap
+		val functions = rosettaFunctions.sortBy[name].generateFunctions(version).replaceTabsWithSpaces
+		result.put(FILENAME,functions)
+		return result;
+	}
+	
+	private def generateFunctions(List<RosettaNamed> functions, String version)  '''
+		
+		«fileComment(version)»
+		package functions	
```

Comment:
again package name should be `org.isda.cdm`

Reply:
cannot use dots in package names